charts/sysbox/templates/sysbox/sysbox-deploy-k8s.yaml

```diff
@@ -0,0 +1,90 @@
+---
+# CHANGES
+# metadata.labels added
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: sysbox-deploy-k8s
+  namespace: kube-system
+  labels:
+    {{- include "sysbox.labels" . | nindent 4 }}
+spec:
+  selector:
+      matchLabels:
+        sysbox-install: "yes"
+  template:
+    metadata:
+        labels:
+          sysbox-install: "yes"
+    spec:
+      serviceAccountName: sysbox-label-node
+      nodeSelector:
+        sysbox-install: "yes"
+      containers:
+      - name: sysbox-deploy-k8s
+        image: registry.nestybox.com/nestybox/sysbox-deploy-k8s
+        imagePullPolicy: Always
+        command: [ "bash", "-c", "/opt/sysbox/scripts/sysbox-deploy-k8s.sh install" ]
```

Comment:
`sysbox-deploy-k8s.sh` did you take a quick peek at what this script does if it's public, just to be safe?

Reply:
that's a good idea, I'll go find it

Reply:
can always docker exec into the image if it's not easily findable in the repos 😄

Reply:
Well, for a bash script, it is surprisingly well written, and it carves out special support for GKE oddities. It LGTM